tools/map/build_external_places.py

```
@@ -0,0 +1,279 @@
+     "郡國志 牂牁郡 治 且蘭. 귀주 황평 비정 채택 — 貴陽설과 미결", "Guizhou", 60),
+    ("蜀郡屬國", "COMMANDERY", 6, "Ya'an", CN, None, None,
+     "郡國志 蜀郡屬國 治 漢嘉. 아안 일대", "Sichuan", 40),
+    ("張掖居延屬國", "COMMANDERY", 6, "Ejin Banner", CN, None, None,
+     "郡國志 張掖居延屬國 治 居延. 額濟納 거연택", "Inner Mongolia", 60),
+
+    # --- 漢 밖 세력. 三國志 魏書30 烏丸鮮卑東夷傳 · 後漢書 卷85 東夷列傳 ---
+    # 중요 세력은 郡治급(lv6)으로 올린다 — 도로 간선의 허브가 되어 실제로 오갈 수 있어야 한다.
+    ("高句麗", "EXTERNAL_PLACE", 6, "Ji'an", CN, None, None,
+     "국내성 = 지린성 지안. 유리왕 3년 천도 이후 도읍", "Jilin", 10),
+    ("夫餘", "EXTERNAL_PLACE", 6, "Nong'an County", CN, None, None,
+     "쑹화강 유역 눙안 일대. 東夷傳 「夫餘在長城之北，去玄菟千里」", None, 20),
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)